### DISCUSSION TITLE
Windows App bootstrap failing

### DISCUSSION THREAD

**Discussion by ElSaico** [pywinrt/pywinrt#67] [Q&A] [answered]
First of all: thank you for release 2.1.0, with it I could finally start to work with the App SDK.

However, the bootstrap module is throwing an error:
```
Traceback (most recent call last):
  File "C:\Users\<snip>\Projects\EDMarketConnector\EDMarketConnector.py", line 2252, in <module>
    from winrt.microsoft.windows.applicationmodel.dynamicdependency import bootstrap
  File "C:\Users\<snip>\Projects\EDMarketConnector\.venv\Lib\site-packages\winrt\microsoft\windows\applicationmodel\dynamicdependency\bootstrap\__init__.py", line 4, in <module>
    from ...... import (
ImportError: DLL load failed while importing _winrt_microsoft_windows_applicationmodel_dynamicdependency_bootstrap: The specified module could not be found.
```

I'm running Windows 10 Enterprise version 22H2.
Also, the reason I did not open an issue straight away because it might be something weird on my machine...

**Comment by dlech** [comment]
Did you install the Windows App Runtime?

Does the example program work? https://github.com/pywinrt/pywinrt/blob/main/samples/win_app_sdk/hello_app.py

**Reply by dlech** [reply, marked as answer]
Oof. Now that I am more awake, I realize what the problem is. The required .dll didn't get copied in the binary wheel that was uploaded to PyPI.

Workaround is to put `Microsoft.WindowsAppRuntime.Bootstrap.dll` from `runtimes/win-x64/native/` in `.venv\Lib\site-packages\winrt\` . 

The DLL can be found in this package: https://www.nuget.org/api/v2/package/Microsoft.WindowsAppSDK/1.5.240311000
